kubeflow/instance/kustomize/kustomization.yaml

```diff
@@ -0,0 +1,4 @@
+kind: Kustomization
+namespace: istio-system
+resources:
+- ../../../upstream/manifests/istio/cluster-local-gateway/base 
```

Comment:
Hi @jal06, if the resource doesn't need any user modification, we can skip putting it in `instance` folder. You can just call `kustomize build` from the upstream folder directly.